trial-config-service/src/main/java/uk/ac/ox/ndph/mts/trial_config_service/model/Person.java

```diff
@@ -0,0 +1,84 @@
+package uk.ac.ox.ndph.mts.trial_config_service.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.hibernate.envers.Audited;
+import org.springframework.data.annotation.CreatedBy;
+import org.springframework.data.annotation.CreatedDate;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.SequenceGenerator;
+import java.time.LocalDateTime;
+
+
+@Entity
+@Audited
+public class Person {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "person_sequence")
+    @SequenceGenerator(name = "person_sequence", sequenceName = "person_sequence", allocationSize = 1)
+    @Column
+    private Long id;
+
+    @Column(unique = true)
+    private String azureOid;
+
+    @JsonIgnore
+    /*
+    This is a dummy relationship - it won't be a direct 1:1 here but will be mediated by Roles.
+    Dummied for the purpose of story 170.
+     */
+    @OneToOne
+    @JoinColumn(name = "trialsite_id")
```

Comment:
```suggestion
    @JoinColumn(name = "trial_site_id")
```

?

Reply:
I think trialsite_id is correct - its the id for a trialSite? We use the following pattern when mapping entites via the id - obj_id - this is used throughout all models - in this case the object is  trialSite.